sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -269,6 +269,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val HIVE_DATASOURCE_PARTITION_PRUNING =
```

Comment:
Why is this `HIVE_DATASOURCE_PARTITION_PRUNING` instead of `HIVE_FILESOURCE_PARTITION_PRUNING`?